src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -2415,4 +2415,10 @@ TableDesignerConfirmationText = I have read the summary and understand the poten
 
 SqlProjectModelNotFound(string projectUri) = Could not find SQL model from project: {0}.
 UnsupportedModelType(string type) = Unsupported model type: {0}.
-GetUserDefinedObjectsFromModelFailed = Failed to get user defined objects from model.
\ No newline at end of file
+GetUserDefinedObjectsFromModelFailed = Failed to get user defined objects from model.
+
+#ObjectManagement Service
+NotAllowedCharacterForRenaming = The name contains characters which are not allowed
```

Comment:
You can remove these validation messages since you're not using them anymore.

Reply:
Yes, I saw it at the moment as I pushed it ;)